app/gzac/README.md

```diff
@@ -1,76 +1,53 @@
-### GZAC
+# GZAC
 
 This is a GZAC - Valtimo edition for local development.
-Includes an array of additional services for OpenZaak/OpenFormulieren
 
-Clone external git repo https://github.com/valtimo-platform/valtimo-docker-profiles
+## Running the Valtimo libraries from source
 
-#### 1 Docker-compose up
+The following instructions are for running Valtimo from source code. Usually with the intention of improving the Valtimo
+libraries. If you want to get started with Valtimo for you own application, please consult
+the [getting started guide](https://docs.valtimo.nl/getting-started/first-dive/creating-your-own-valtimo-implementation).
 
-Run in terminal:
-<code>valtimo-docker-profiles/gzac-platform% docker-compose up</code>
+### Prerequisites:
 
-#### 2 Run Spring-boot-application
+- Java 17
+- [Git](https://git-scm.com/downloads)
+- An IDE like [IntelliJ](https://www.jetbrains.com/idea/download/) or [Eclipse](https://www.eclipse.org/downloads/)
+- [Docker (Desktop)](https://www.docker.com/products/docker-desktop/)
 
-Run in IntellJ Gradle: <code>app -> gzac -> application -> bootRun</code>
+### Start docker containers for supporting services
+
+Make sure docker is running.
+
+Run IntellJ Gradle task <code>app -> gzac -> Tasks -> docker -> composeUpGzac</code>.
 
 Or
 
-Run in terminal: <code>app/gzac% ../../gradlew bootRun</code>
+Open a terminal and run command
+```
+./gradlew :app:gzac:composeUpGzac
```

Comment:
This looks like a step we can automate when running `bootRun` see my comment on `dockerComposeGzac.gradle`